app/views/green_lanes/category_assessments/create.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= page_header "Search results for &lsquo;#{h @category_assessments_search.commodity_code}&rsquo;".html_safe %>
+<%= page_header "Search results for #{h @category_assessments_search.commodity_code}" %>
```

Comment:
Not a problem but more FYI, the manual escaping of comm code (the `h`) can get dropped because since the string is no longer being marked `.html_safe` so the whole string will get escaped automatically